tests/test_policy.py

```diff
@@ -262,6 +262,10 @@ def test_securityhub_resource_support(self):
             'AwsEc2TransitGateway',
             'AwsEcsContainer',
             'AwsEcsTask',
+            'AwsBackupRecoveryPoint',
+            # https://github.com/cloud-custodian/cloud-custodian/issues/7775
```

Comment:
Opened an issue to add support for backup-plan here: https://github.com/cloud-custodian/cloud-custodian/issues/7775